js/menu.js

```diff
@@ -939,6 +939,37 @@ function makeLinkToId(id) {
   return (targetSec === 'index' ? './' : targetSec + '.html') + hash;
 }
 
+function doShortcut(e) {
+  if (!(e.target instanceof HTMLElement)) {
+    return;
+  }
+  let target = e.target;
+  let name = target.nodeName.toLowerCase();
+  if (name === 'textarea' || name === 'input' || name === 'select' || target.isContentEditable) {
+    return;
+  }
+  if (e.key === 'm' && !e.altKey && !e.ctrlKey && !e.metaKey && !e.shiftKey && usesMultipage) {
+    let pathParts = location.pathname.split('/');
+    let hash = location.hash;
+    if (pathParts[pathParts.length - 2] === 'multipage') {
```

Comment:
Is there a less fragile test? This appears to get tripped up on trailing slashes.

Reply:
What do you mean by "tripped up"? The only valid paths are `multipage/` and `multipage/some-file.html`, and

```js
let pathParts = 'https://tc39.es/ecma262/multipage/'.split('/');
pathParts[pathParts.length - 2] === 'multipage'; // true
```

```js
let pathParts = 'https://tc39.es/ecma262/multipage/index.html'.split('/');
pathParts[pathParts.length - 2] === 'multipage'; // true
```

`multipage` is a directory name, so the trailing slash is expected.

Reply:
`multipage//`?

Reply:
Not a valid url; see for yourself: https://tc39.es/ecma262/multipage//